### ISSUE TITLE
ameloblast develops from

### ISSUE THREAD

**Issue by cerivs** [cerivs/zebrafish-anatomical-ontology#97]
User input: odontoblasts form from neural crest cells but ameloblasts develop from either stomodeal ectoderm or pharyngeal endoderm. 

References for zebrafish:
Huysseune, A., van der Heyden, C., and Sire, J.-Y. (1998). Early development of the zebrafish (Danio rerio) pharyngeal dentition (Teleostei, Cyprinidae). Anat Embryol, 198, 289–305.

This paper doesn't go into the origin per se, but it describes ameloblasts as originating from the dental epithelium, which in the case of zebrafish originates from the pharyngeal endoderm. There haven't been careful fate-mapping studies of this, but it fits in with years of data from mammals

Ref. for mammals:
de Beer, G. (1947). The differentiation of neural crest cells into visceral cartilages and odontoblasts in Amblystoma, and a re-examination of the germ-layer theory. Proc R Soc Lond, B, Biol Sci, 134, 377–398.

Soukup, V., Epperlein, H.-H., Horácek, I., and Cerny, R. (2008). Dual epithelial origin of vertebrate oral teeth. Nature, 455, 795–798.

So basically ameloblasts arise from the inner dental epithelium which in vertebrates can arise either from stomodeal ectoderm or pharyngeal endoderm. Zebrafish don't have oral teeth, so their ameloblasts should always arise from pharyngeal endoderm. In all vertebrate teeth, the odontoblasts that make dentin arise from cranial neural crest.

Original comment by: ybradford
